plugins/webp-uploads/picture-element.php

```diff
@@ -148,6 +148,16 @@ function webp_uploads_wrap_image_in_picture( string $image, string $context, int
 					esc_attr( $image_srcset ),
 					is_string( $sizes ) ? sprintf( ' sizes="%s"', esc_attr( $sizes ) ) : ''
 				);
+			} else {
+				// Fallback when wp_calculate_image_srcset returns false - generate simple source without srcset.
```

Comment:
The `SOURCE` still has a `srcset` (as it can't have `src`).

```suggestion
				// Fallback when wp_calculate_image_srcset returns false - generate simple non-responsive srcset.
```